### ISSUE TITLE
Better error when the scaffold doesn't exist

### ISSUE THREAD

**Issue by coderarity** [flatiron/flatiron#55]
Currently, if you try to use `flatiron create` with a scaffold that doesn't exist, it crashes and gives you ugly error output. For example, try `flatiron create test blog`, which will produce a huge error output. Instead, it should say something like 'Scaffold does not exist', and give the user a list of available scaffolds. Perhaps a separate command should be made that lists out the available scaffolds, and in the error for creating a scaffold that doesn't exist, tell the user to run this command.

tl;dr - have a way to list scaffolds and show a proper error message when the user tries to create a scaffold that doesn't exist
